code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -41,6 +41,8 @@ GLOBAL_LIST_EMPTY(antagonists)
 	var/clown_text_span_class = "boldnotice"
 	/// The url page name for this antagonist, appended to the end of the wiki url in the form of: [GLOB.configuration.url.wiki_url]/index.php/[wiki_page_name]
 	var/wiki_page_name
+	/// The organisation, if any, this antag is associated with
+	var/datum/antag_org/organisation
```

Comment:
I don't think this should be on all antags, just traitors. Just so things are properly encapsulated. If someone wants to add this to clings/vampires, they can move it down themselves

Reply:
Moving this down to the traitor antag would require refactoring the on_gain proc so the introduction message is shown properly to an unnecessary extent.

Reply:
I think this could be made use of by more than just traitors.
We could add vampire cabals to the lore, cling dynasties, wizard schools, etc.

Reply:
Off topic here, but since this is on the same line. Variables declared should use American English as the spelling, in accordance with `CONTRIBUTING.md`

Reply:
All instances have been rewritten to 'Organization' per the guidelines, thank you for pointing this out!